src/metabase/cmd/copy.clj

```diff
@@ -97,7 +97,10 @@
     :model/TablePrivileges
     :model/AuditLog
     :model/RecentViews
-    :model/UserParameterValue]
+    :model/UserParameterValue
+    ;; 49+
+    :model/Notification
```

Comment:
I'm not sure if we want to copy it? putting it here bc we copied Pulse. but I made a note in the epic to review this later.